pkg/proxy/iptables/proxier_test.go

```diff
@@ -380,7 +76,7 @@ const testNodeIP = "192.168.0.2"
 const testExternalClient = "203.0.113.2"
 const testExternalClientBlocked = "203.0.113.130"
 
-func NewFakeProxier(ipt utiliptables.Interface) *Proxier {
+func NewFakeProxier(ipt utiliptables.Interface, execer exec.Interface) *Proxier {
```

Comment:
why do we need this change?

Reply:
I had to do this to accommodate the TestProxierDeleteNodePortStaleUDP test.
https://github.com/kubernetes/kubernetes/blob/12ceec47aa6e38872fe03343aba2cb957f3e8e61/pkg/proxy/iptables/proxier_test.go#L5406-L5408

The test depends on assertions on fake exec, I had to pass this built-up fexec to the `pkg/proxy/conntrack`